main.cc

```diff
@@ -152,6 +144,31 @@ int main(){
 	return 0;
 }
 
+std::string getToken() {
+	std::string token;
+	
+		/*
+			First attempt to read the token from the BOT_TOKEN environment variable.
+		*/
+	char const* env = std::getenv("BOT_TOKEN");
+	if(env != nullptr) {
+		token = (std::string) env;
```

Comment:
Wouldn't it be safer to use `std::string(env)`?

Reply:
I'm not sure it matters much here, env should contain a legitimate series of chars if it pass the null-check

Reply:
I'd still like this change; it's just considered better form.